src/popup/i18n/locales/de.json5

```diff
@@ -0,0 +1,162 @@
+        // Description for appearance
+        description: "Ändere die Erscheinung. Änderungen werden nur auf dem Browser angewendet",
+        // Supported appearances
+        appearances: {
+          // The theme follows the current OS setting. This is the default selection
+          device: "Geräte Theme",
+          // Default light theme which is designed for daily use
+          light: "Helles Theme",
+          // A theme which comes with a contrast not so strong as black-on-white and which best fits to night use
+          dark: "Verdunkelt Theme"
+        }
+      },
+      // Prompt users to change advanced settings
+      advanced: "Erweiterte Einstellungen",
+      // Display the current version
+      version: "Version: v{version}",
+      // Describe the backend engine of this project
+      engine: {
+        // Template used to construct the description for the engine
+        label: "Powered by {link}",
+        // Name of the engine. There is no need to translate this key
+        value: "Non-stop-story",
+        // Link to the engine's homepage. There is no need to translate this key
+        href: "https://github.com/YunzheZJU/non-stop-story"
+      },
+      // Description prepended to the contact email
+      contact: {
+        // Template used to construct the description for the engine
+        label: "Bei Fragen, kontaktieren Sie {link}",
+        // Name of the engine. There is no need to translate this key
+        value: "help@holo.dev",
+        // Link to the engine's homepage. There is no need to translate this key
+        href: "mailto:@:app.settings.contact.value"
+      }
+    },
+    // Pull-to-load
+    pullToLoad: {
+      // Prompt users to pull down the list to load new data
+      hint: "Ziehe um mehr zu laden",
+      // Indicate that the loading is being performed
+      loading: "Lade",
+      // Indicate that the loading is done successfully
+      success: "Erfolgt",
+      // Indicate that all available data has been loaded
+      finished: "Abgeschlossen"
+    }
+  },
+  // Component LiveList
+  liveList: {
+    lives: {
+      // Lives that has ended
+      ended: {
+        label: "Beendete Streams",
+        loading: "Lade beendete Streams"
+      },
+      // Lives that are undergoing
+      current: {
+        label: "Live",
+        loading: "Lade laufende Streams"
+      },
+      // Lives that are scheduled to perform
+      scheduled: {
+        label: "Geplante Streams",
+        loading: "Lade geplante Streams"
+      }
+    }
+  },
+  // Component LiveListEnded
+  liveListEnded: {
+    // Used to customize dayjs#calendar output
+    calendar: {
+      // A token string representing the format when displaying datetime with dayjs.
+      // You are encouraged to adjust the text wrapped in '[' and ']' or its position.
+      // See also: https://day.js.org/docs/display/format
+      sameDay: "[Heute]",
+      lastDay: "MMM Do, [Gestern]",
+      lastWeek: "MMM Do, dddd",
```

Comment:
Maybe the `,` is redundant here? I'm not sure.

![0](https://user-images.githubusercontent.com/23452609/114742976-cfa3f100-9d7e-11eb-93c8-2e7d2447ea91.png)

Reply:
![1PdrqiyZTe](https://user-images.githubusercontent.com/80579142/114753281-92783880-9d57-11eb-8405-34e71d89b253.png)

Weird, i don't have that issue. is that maybe because of the Browser?

Reply:
I removed the `,` now from the json

Reply:
yes the time display is still in English because I missed the de locale of dayjs. It should render improperly  as my screenshot shows, in Chrome at least, as long as we have added that line.